### ISSUE TITLE
Accessibility Guidelines - image under 1.2 Metadata processing not visible

### ISSUE THREAD

**Issue by chrisONIX** [w3c/publ-a11y#714]
In the published version of Accessibility Metadata Display Guide for Digital Publications 2.0 - section  1.2 Metadata processing - the image is not visible on Chrome or Safari.

**Comment by mattgarrish** [comment]
Yes, looks like it got missed in the release files. I've created a pull request to have it added: https://github.com/w3c/cg-reports/pull/62
